test/test_dgmulti_1d.jl

```diff
@@ -25,7 +25,8 @@ isdir(outdir) && rm(outdir, recursive = true)
         t = sol.t[end]
         u_ode = sol.u[end]
         du_ode = similar(u_ode)
-        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000
+        Trixi.rhs!(du_ode, u_ode, semi, t) # run once first to deal with spurious allocations
+        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 5000
```

Comment:
I'm curious about how is this value determined? Like why 5000 here and 1000 before

Reply:
Not sure; @DanielDoehring?

Reply:
So if the type is unstable, the memory allocation size will change by orders of magnitude? @DanielDoehring

Reply:
Yeah pretty much :+1:

Reply:
I see 👍